libs-scala/db-utils/src/main/scala/com/digitalasset/http/dbbackend/DBConfig.scala

```diff
@@ -84,88 +92,64 @@ abstract class ConfigCompanion[A, ReadCtx](name: String) {
 }
 
 object JdbcConfig
-    extends ConfigCompanion[JdbcConfig, DBConfig.SupportedJdbcDriverNames]("JdbcConfig")
+    extends ConfigCompanion[JdbcConfig, DBConfig.JdbcConfigDefaults]("JdbcConfig")
     with StrictLogging {
 
   final val MinIdle = 8
   final val IdleTimeout = 10000L // ms, minimum according to log, defaults to 600s
   final val ConnectionTimeout = 5000L
 
-  implicit val showInstance: Show[JdbcConfig] = Show.shows(a =>
-    s"JdbcConfig(driver=${a.driver}, url=${a.url}, user=${a.user}, start-mode=${a.dbStartupMode})"
-  )
+  @scala.deprecated("do I need this?", since = "SC")
+  implicit val showInstance: Show[JdbcConfig] =
+    Show.shows(a => s"JdbcConfig(driver=${a.driver}, url=${a.url}, user=${a.user})")
 
-  def help(implicit supportedJdbcDriverNames: DBConfig.SupportedJdbcDriverNames): String =
+  def help(otherOptions: String = "")(implicit jcd: DBConfig.JdbcConfigDefaults): String =
     "Contains comma-separated key-value pairs. Where:\n" +
-      s"${indent}driver -- JDBC driver class name, ${supportedJdbcDriverNames.unwrap.mkString(", ")} supported right now,\n" +
+      s"${indent}driver -- JDBC driver class name, ${jcd.supportedJdbcDrivers.mkString(", ")} supported right now,\n" +
       s"${indent}url -- JDBC connection URL,\n" +
       s"${indent}user -- database user name,\n" +
       s"${indent}password -- database user password,\n" +
-      s"${indent}tablePrefix -- prefix for table names to avoid collisions, empty by default,\n" +
-      s"${indent}createSchema -- boolean flag, if set to true, the process will re-create database schema and terminate immediately. This is deprecated and replaced by start-mode, however if set it will always overrule start-mode.\n" +
```

Comment:
I'm missing what happened here. Why were these dropped?

Reply:
Thanks to your comment I rechecked myself and noticed that reintroduced the `createSchema` field.

So far I see the things in there moved to `ledger-service/http-json-cli/src/main/scala/com/daml/http/dbbackend/DBConfig.scala`

Reply:
This file is common to trigger service and json-api. So documenting (and parsing, for that matter) here advertises that trigger service supports these options. I included this note in the changelog, see PR description.